soc/ti/k3/am6x/Kconfig.defconfig

```diff
@@ -16,11 +19,13 @@ config NUM_IRQS
 	int
 	default 64 if SOC_SERIES_AM6X_M4
 	default 280 if SOC_SERIES_AM6X_A53
+	default 512 if SOC_SERIES_AM6X_R5
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	int
 	default 400000000 if SOC_SERIES_AM6X_M4
 	default 200000000 if SOC_SERIES_AM6X_A53
+	default 19200000 if SOC_SERIES_AM6X_R5
```

Comment:
Seems to me as if this is not generally true. For the AM65x, 200000000 give accurate timing, 19200000 is way too fast there.

Reply:
As this will change for different SoCs, perhaps this info belongs in the SoCs DT? I've seen something like this done elsewhere:

```
config SYS_CLOCK_HW_CYCLES_PER_SEC
	default $(dt_node_int_prop_int,/cpus/cpu@0,clock-frequency)
```

Reply:
Also the J721e should be affected by https://github.com/zephyrproject-rtos/zephyr/blob/8068cb2567508d760e71f04042f0301fdd081403/drivers/timer/ti_dmtimer.c#L26
Or do you have `CONFIG_TIMER_READS_ITS_FREQUENCY_AT_RUNTIME` enabled?

Reply:
The setting here will depend on which clock is feeding the DM timer you are using. It is not the arch timer like with M4. Since the clock is delectable at runtime, once we get TI-SCI clock support in we will enable `CONFIG_TIMER_READS_ITS_FREQUENCY_AT_RUNTIME`.

Reply:
I would avoid `CONFIG_TIMER_READS_ITS_FREQUENCY_AT_RUNTIME` unless really necessary. Comment from a kernel maintainer:

![Screenshot_20241005-172133](https://github.com/user-attachments/assets/c4b6c444-7f39-4bfa-96c5-bf679a563279)

Reply:
So, what to do with this setting for now? Leave it and revisit when adding more SoCs under `SOC_SERIES_AM6X_R5`?

Reply:
I'd like to just revisit this when we add SOC_AM6548_R5.